circuit_setup/circuits-mdl/main.circom

```diff
@@ -169,6 +167,7 @@ template Main(max_msg_bytes, max_json_bytes, field_byte_len, n, k) {
     // Now we've confirmed that dob_preimage is authenticated: parse out the YYYY-MM-DD 
     // and confirm it equals dob_value
 
+    // Copy dob_preimage into a local array 'a' // FIXME: why not use dob_preimage directly?
```

Comment:
I think I was trying to keep the code on lines 174-176 more compact. This will probably be re-written anyway when we support variable length digest IDs.

Reply:
Removed the FIXME